gcp-deployer/generate-yaml.sh

```diff
@@ -36,9 +38,16 @@ while test $# -gt 0; do
         esac
 done
 
-if [ $OVERWRITE -ne 1 ] && [ -f $GENERATED_FILE ]; then
-  echo File $GENERATED_FILE already exists. Delete it manually before running this script.
+if [ $OVERWRITE -ne 1 ] && [ -f $MACHINE_GENERATED_FILE ]; then
+  echo File $MACHINE_GENERATED_FILE already exists. Delete it manually before running this script.
   exit 1
 fi
 
-sed -e "s/\$GCLOUD_PROJECT/$GCLOUD_PROJECT/" $TEMPLATE_FILE | sed -e "s/\$ZONE/$ZONE/" > $GENERATED_FILE
+sed -e "s/\$ZONE/$ZONE/" $MACHINE_TEMPLATE_FILE > $MACHINE_GENERATED_FILE
+
+if [ $OVERWRITE -ne 1 ] && [ -f $CLUSTER_GENERATED_FILE ]; then
```

Comment:
Consider moving this up above any modifications of either the machines.yaml or cluster.yaml file so that the script doesn't generate the machines.yaml file and then fail on the cluster.yaml file. In that scenario, after the user deletes the cluster.yaml file and runs the script again, the script would fail with an error message the the machines.yaml file already exists.